api/Attr.json

```diff
@@ -0,0 +1,166 @@
+{
+  "api": {
+    "Attr": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Attr",
+        "support": {
+          "webview_android": {
+            "version_added": true,
+            "notes": "As of Chrome 45, this property no longer inherits from Node."
+          },
+          "chrome": {
+            "version_added": true,
+            "notes": "As of Chrome 45, this property no longer inherits from Node."
+          },
+          "chrome_android": {
+            "version_added": true,
+            "notes": "As of Chrome 45, this property no longer inherits from Node."
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": true
+          },
+          "ie": {
+            "version_added": true
+          },
+          "opera": {
+            "version_added": true,
+            "notes": "As of Opera 32, this property no longer inherits from Node."
+          },
+          "opera_android": {
+            "version_added": true,
+            "notes": "As of Opera 32, this property no longer inherits from Node."
+          },
+          "safari": {
+            "version_added": true
+          },
+          "safari_ios": {
+            "version_added": true
+          }
+        }
+      },
```

Comment:
This compat object and the subfeatures should have status info like this:

```json
"status": {
  "experimental": false,
  "standard_track": true,
  "deprecated": false
}
```

For example:

https://github.com/mdn/browser-compat-data/blob/ac286bac1a3e418d03f3e324907fdaaeb4e22dc0/api/AbortController.json#L47-L51